src/back/auth/src/repo/table_session.hpp

```diff
@@ -25,6 +25,15 @@ class Session final {
 
 	void MarkInactive(const model::Session& s);
 
+	void Refresh(
+		const model::Session& data,
+		const boost::uuids::uuid& oldSessionId
+	);
+
+	void MarkInactiveAndCleanTokensById(const boost::uuids::uuid& sid);
+
+	void BlockEverySessionByUser(const std::string& userId);
+
```

Comment:
Давай соблюдать порядок перечисления функций таким как он идёт в файле определений. Так их проще искать.
```suggestion
	void BlockEverySessionByUser(const std::string& userId);
	void Refresh(
		const model::Session& data,
		const boost::uuids::uuid& oldSessionId
	);

	void MarkInactiveAndCleanTokensById(const boost::uuids::uuid& sid);


```

Reply:
Поправил (но в других файлах пока не проверял, чтобы порядок был везде правильным).